docs/advisors/checks/mysql_configuration_max_connections_usage.md

```diff
@@ -0,0 +1,12 @@
+# MySQL max connection usage check
+## Description
+The status variable "max_used_connections" indicates the highest number of connections utilized since the previous system restart. If this value approaches the predetermined "max_connections" limit, it warrants attention.
```

Comment:
```suggestion
The status variable **max_used_connections** indicates the highest number of connections utilized since the previous system restart. If this value approaches the predetermined **max_connections** limit, it warrants attention.
```